Update Readme and add Taskfile for setup.

amjustin13 opened the pull request:
I have also fixed what seemed to be a typo on nvm.go that prevented me from running tests.

Signed-off-by: Amanda Justiniano-Pagan <amjustin@boro-6.boro.hpdd.intel.com>

Commit:
Update Readme and add Taskfile for setup.

I have also fixed what seemed to be a typo on nvm.go that prevented me from running tests.

Signed-off-by: Amanda Justiniano-Pagan <amjustin@boro-6.boro.hpdd.intel.com>

amjustin13 commented:
I am working on adding a few things and still experimenting with the ipmctl lib and the test that is currently in the repo. I will create a new pull request for that work.

tanabarr commented on README.md:
```diff
@@ -1,2 +1,41 @@
+
 Go bindings for the libipmctl native C lib to enable management of DCPMMs from Go source 
+
+This is a Go interface for [ipmctl](https://github.com/intel/ipmctl) which is also a work in progress.
```
don't think you need "which is also a work in progress"

amjustin13 replied:
Agreed. Will change.

tanabarr commented on README.md:
```diff
@@ -1,2 +1,41 @@
+
+This is a [Go](https://golang.orghttps://golang.org/doc/install)
+project, so the Go development tools are naturally required. We
+recommend the most current Go release available. As of July 2018, the project has been built and tested with Go 1.9.
```
we could update this to go 1.11.5 which is what we are currently testing in CI

amjustin13 replied:
Agreed. Will change.

tanabarr commented on Taskfile.yml:
```diff
@@ -0,0 +1,23 @@
+    cmds:
+      - go test -v
+
+  build-spdk:
```
needs to be changed from spdk?

amjustin13 replied:
Yes it does. Good catch, thank you!

Commit:
Updating some redundancy in Readme and typo in Taskfile.

Changed the go version in the documentation to 1.11.5, remove some redundant information and updated the Taskfile to reflect the appropriate build task.

Signed-off-by: Amanda Justiniano <amanda.justiniano-pagn@intel.com>
